com.github.unrud.djpdf.yaml

```diff
@@ -891,6 +891,17 @@ modules:
 
   - shared-modules/openjpeg/openjpeg.json
 
+  # libtiff with JPEG support
+  # https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1117
+  - name: libtiff
+    buildsystem: cmake-ninja
+    cleanup:
+       - '*.so'
```

Comment:
I don't understand the use of this?
It removes `/app/lib/libtiff.so` and `/app/lib/libtiffxx.so`. They are symlinks to `libtiff.so.5` and `libtiffxx.so.5`.

Reply:
This is common practice. `.so` are only for development: the linker look for the .so and the resolution of the symlink is used to link to the current ABI version.

Reply:
And I didn't put it in the top level because I don't have time to figure out if there is any unversionned `.so` like it happen with plugins.

Reply:
Thanks for the explanation.